libc/sigaction.c

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Given the following set of files are commented out from libc/CMakeLists.txt, do you still need to include them as part the PR?

   #${MUSLSRC}/signal/sigaction.c
    #${MUSLSRC}/signal/sigaddset.c
    #${MUSLSRC}/signal/sigandset.c
    #${MUSLSRC}/signal/sigdelset.c
    #${MUSLSRC}/signal/sigemptyset.c
    #${MUSLSRC}/signal/sigfillset.c
    #${MUSLSRC}/signal/sigignore.c
    #${MUSLSRC}/signal/sigisemptyset.c
    #${MUSLSRC}/signal/sigismember.c
    #${MUSLSRC}/signal/sigorset.c
    #${MUSLSRC}/signal/sigset.c

Reply:
Good idea. I removed these.